Automattic/vip-design-system
# FORNO-1690: Add toggle-group component

## Description

Add toggle group component.
[Click for design](https://www.figma.com/design/mLgFfiIjPIEMRQrfrJyDiN/Insights-%26-Metrics?node-id=1074-5105&t=XbenphK4b7Po1yqB-0).

<img width="236" alt="Screenshot 2024-06-24 at 12 38 45 AM" src="https://github.com/Automattic/vip-design-system/assets/6948790/e2935be5-d1c9-4c6f-b66f-7c877e2561b8">


## Checklist


## Steps to Test

1. Pull down PR.
1. `npm run dev`.
1. Open Storybook.
1. Eat cookies.
1. Verify cookies are delicious.
